### ISSUE TITLE
npm registry error 'no valid versions available'

### ISSUE THREAD

**Issue by malua** [passslot/cordova-plugin-passbook#11]
hello,

somehow the npm entry for this plugin isn't valid anymore. `npm install` produces this error:
`
fetchPackageMetaData error for com.passslot.cordova.plugin.passbook@~0.2.1 No valid versions available for com.passslot.cordova.plugin.passbook`

Is there still somebody maintaining this repo? otherwise i would maybe fork it and make a new npm registry entry

**Comment by malua** [comment]
maybe the problem is that there are 2 entries for this plugin in package.json.

one is `cordova-plugin-passbook`, the other is `com.passslot.cordova.plugin.passbook`

**Comment by passslot** [comment]
We will have look into this.

We think you should still be able to use the plugin when you add it directly from github via 
cordova plugin add https://github.com/passslot/cordova-plugin-passbook

If you already found a solution, we would be happy for a pull request

**Comment by jeremychauvet** [comment]
Any update on this issue ?

**Comment by Ashygoyal** [comment]
Hi @malua , were you able to achieve the functionality?

**Comment by malua** [comment]
Yes, adding it directly via Github works! just use the the command passslot has posted. Unfortunately I don't have time to look at the issue and resolve it at the moment

**Comment by mrbowen724** [comment]
Adding it via Github does indeed, work, but not out of the box. I had to make some manual updates to config.xml and package.json to get this to work, as well as build in ionic.

**package.json**:
"dependencies": {
...
"cordova-plugin-passbook": "https://github.com/passslot/cordova-plugin-passbook.git",
},
"**plugins**": {
...
  "cordova-plugin-passbook": {}
}

config.xml:
<plugin name="cordova-plugin-passbook" spec="~0.2.1" />
